enterprise/cmd/executor/internal/config/config.go

```diff
@@ -179,8 +181,18 @@ func getKubeConfigPath() string {
 }
 
 func (c *Config) Validate() error {
-	if c.QueueName != "" && c.QueueName != "batches" && c.QueueName != "codeintel" {
+	if c.QueueName == "" && c.QueueNames == "" {
+		c.AddError(errors.New("One of EXECUTOR_QUEUE_NAME and EXECUTOR_QUEUE_NAMES should be set"))
+	} else if c.QueueName != "" && c.QueueNames != "" {
+		c.AddError(errors.New("Only one of EXECUTOR_QUEUE_NAME and EXECUTOR_QUEUE_NAMES should be set"))
```

Comment:
Same here, slight change (I don't really like my suggestion, but something to that gist)

```suggestion
		c.AddError(errors.New("Only set either EXECUTOR_QUEUE_NAME or EXECUTOR_QUEUE_NAMES"))
```